frontend/src/app/core/services/playlist.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from '@angular/core';
+import { environment } from "../../../environments/environment";
+import { HttpClient, HttpParams } from "@angular/common/http";
+import { PostPlaylistDto } from '../models/post-playlist-dto';
+import { getHttpOptionsWithAuthenticationHeader } from "../functions/get-http-options-with-authorization-header";
+import { UserPlaylistsDto } from '../models/user-playlists-dto';
+import { Observable } from 'rxjs';
+import { PlaylistVideosDto } from '../models/playlist-videos-dto';
+
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PlaylistService {
+  private readonly videoPageWebAPIUrl: string = `${environment.webApiUrl}`;
+
+  constructor(private httpClient: HttpClient) { }
+
+  postPlalist(postPlaylist: PostPlaylistDto): Observable<void>{
+    const httpOptions = {
+      headers: getHttpOptionsWithAuthenticationHeader().headers
+    };
+
+    return this.httpClient.post<void>(`${this.videoPageWebAPIUrl}/playlist/details`, postPlaylist, httpOptions);
```

Comment:
i thing instead of writing httpOptions we can just have getHttpOptionsWithAuthenticationHeader()

Reply:
I don't like it. This way every service looks excatly the same so it's easier to understand